src/CMakeLists.txt

```diff
@@ -73,40 +73,113 @@ ADD_DEPENDENCIES (rbprm-corba generate_idl_cpp)
 ADD_DEPENDENCIES (rbprm-corba generate_idl_python)
 TARGET_INCLUDE_DIRECTORIES(rbprm-corba PUBLIC ${CDD_INCLUDE_DIRS})
 
-INSTALL(
-  FILES
-  ${CMAKE_CURRENT_SOURCE_DIR}/hpp/corbaserver/rbprm/__init__.py
-  DESTINATION ${PYTHON_SITELIB}/hpp/corbaserver/rbprm
-  )
-INSTALL(
-  FILES
-  ${CMAKE_CURRENT_SOURCE_DIR}/hpp/corbaserver/rbprm/client.py
-  ${CMAKE_CURRENT_SOURCE_DIR}/hpp/corbaserver/rbprm/rbprmbuilder.py
-  ${CMAKE_CURRENT_SOURCE_DIR}/hpp/corbaserver/rbprm/rbprmfullbody.py
-  ${CMAKE_CURRENT_SOURCE_DIR}/hpp/corbaserver/rbprm/rbprmstate.py
-  ${CMAKE_CURRENT_SOURCE_DIR}/hpp/corbaserver/rbprm/state_alg.py
-  ${CMAKE_CURRENT_SOURCE_DIR}/hpp/corbaserver/rbprm/fewstepsplanner.py
-  DESTINATION ${PYTHON_SITELIB}/hpp/corbaserver/rbprm
-  )
-INSTALL(
-  FILES
-  ${CMAKE_CURRENT_SOURCE_DIR}/hpp/corbaserver/rbprm/tools/__init__.py
-  ${CMAKE_CURRENT_SOURCE_DIR}/hpp/corbaserver/rbprm/tools/generateROMs.py
-  ${CMAKE_CURRENT_SOURCE_DIR}/hpp/corbaserver/rbprm/tools/com_constraints.py
-  ${CMAKE_CURRENT_SOURCE_DIR}/hpp/corbaserver/rbprm/tools/plot_analytics.py
-  ${CMAKE_CURRENT_SOURCE_DIR}/hpp/corbaserver/rbprm/tools/affordance_centroids.py
-  ${CMAKE_CURRENT_SOURCE_DIR}/hpp/corbaserver/rbprm/tools/cwc_trajectory.py
-  ${CMAKE_CURRENT_SOURCE_DIR}/hpp/corbaserver/rbprm/tools/cwc_trajectory_helper.py
-  ${CMAKE_CURRENT_SOURCE_DIR}/hpp/corbaserver/rbprm/tools/path_to_trajectory.py
-  ${CMAKE_CURRENT_SOURCE_DIR}/hpp/corbaserver/rbprm/tools/obj_to_constraints.py
-  ${CMAKE_CURRENT_SOURCE_DIR}/hpp/corbaserver/rbprm/tools/time_out.py
-  ${CMAKE_CURRENT_SOURCE_DIR}/hpp/corbaserver/rbprm/tools/display_tools.py
-  DESTINATION ${PYTHON_SITELIB}/hpp/corbaserver/rbprm/tools
-  )
 # Stand alone corba server
 ADD_EXECUTABLE (hpp-rbprm-server hpp-rbprm-corba.cc)
 
 PKG_CONFIG_USE_DEPENDENCY(hpp-rbprm-server hpp-corbaserver)
 PKG_CONFIG_USE_DEPENDENCY(hpp-rbprm-server hpp-pinocchio)
 
 INSTALL (TARGETS hpp-rbprm-server DESTINATION ${CMAKE_INSTALL_BINDIR})
+
+# Install python files : 
```

Comment:
To apply a bit more of "Don't Repeat Yourself" here, you could do:
```cmake
SET({PROJECT_NAME}_PYTHON_FILES
  __init__.py
  __main__.py
  …
  state_alg.py
  )

SET({PROJECT_NAME}_PYTHON_TOOLS
  affordance_centroids.py
  …
  time_out.py
  )

…

FOREACH(file ${${PROJECT_NAME}_PYTHON_FILES})
  PYTHON_INSTALL_ON_SITE("hpp/corbaserver/rbprm" ${file})
ENDFOREACH(file ${${PROJECT_NAME}_PYTHON_FILES})

FOREACH(file ${${PROJECT_NAME}_PYTHON_TOOLS})
  PYTHON_INSTALL_ON_SITE("hpp/corbaserver/rbprm/tools" ${file})
ENDFOREACH(file ${${PROJECT_NAME}_PYTHON_TOOLS})

…
```

But as you have 5 similar blocks, it might not look really better.
As you prefer :)